### DISCUSSION TITLE
How can I get the Dockerfile of the image that slimed by `docker-slim`

### DISCUSSION THREAD

**Discussion by HJX-zhanS** [slimtoolkit/slim#672] [Q&A]
I need to get the `Dockerfile` of the image that slimed by the `docker-slim` tool.

When I used `xray` to analyze before and after slimming, I can extract the files that only existed in the image before slimming from their respective `slim.report.json` files.

However, the total size of these files is not equal to the image size before slimming minus the image size after slimming. Why?

So how can I get a `Dockerfile` of the slimed image?

Thanks!

**Comment by kcq** [comment]
the default build engine doesn't produce a Dockerfile for the minified image, but you can change it to the `docker` build engine (using the filag I mentioned in the other discussion thread)
